events/equeue/equeue_platform.h

```diff
@@ -112,7 +112,7 @@ typedef struct equeue_sema {
     bool signal;
 } equeue_sema_t;
 #elif defined(EQUEUE_PLATFORM_MBED) && defined(MBED_CONF_RTOS_PRESENT)
-typedef unsigned equeue_sema_t[8];
+typedef unsigned equeue_sema_t[9];
```

Comment:
Ah yeah, equeue api is C, but I used mbed's C++ rtos api since I thought that was the portable layer (there is a compile time assert that catches if the class does not fit).

Feel free to change to cmsis rtos api if we're garunteeing the portability of that layer.